src/pluto/pluto.py

```diff
@@ -421,111 +322,319 @@ def generate_graph(v1, v2):
+
+        create_aggregate(data["df_aggregate"], v1, v2, v3, condo, mapped)
+
+        st.header("Source Data Versions")
+        code = st.checkbox("code")
+        if code:
+            st.code(data["version_text"])
+        else:
+            st.markdown(data["version_text"])
+
+        # EXPECTED VALUE
+        st.header("Expected Value Comparison")
+        st.write(
+            "if nothing showed up, then it means there aren't any expected value change"
         )
-        st.plotly_chart(fig)
-        st.write(df)
+
+        create_expected(data["df_expected"], v1, v2)
+
+    def manual_corrections_report(data):
+        def create_corrections_section(df, version, applied):
+            def title_text(version, applied):
+                applied_text = "Applied Manual Corrections " if applied else "Manual Corrections Not Applied "
+                version_text = "for All Versions by Field" if version == "All" else f"introduced in Version {version} by Field"
+
+                return applied_text + version_text
+
+            def filter_by_version(df, version):
+                if version == 'All':
+                    return df
+                else:
+                    return df.loc[df['version'] == version]
+
+            def generate_graph(v1, version, applied):
+                return px.bar(
+                    v1, 
+                    x='field',
+                    y='size', 
+                    text='size',
+                    title=title_text(version, applied), 
+                    labels={'size': 'Count of Records', 'field': 'Altered Field'}
+                )
+
+            def display_corrections_df(corrections):
+                corrections = corrections.sort_values(by=['version', 'reason', 'bbl'], ascending=[False, True, True])
+
+                AgGrid(corrections)
+
+            def field_correction_counts(df):
+                return df.groupby(['field']).size().to_frame('size').reset_index()
+            
+            def empty_message(applied, version):
+                if applied:
+                    st.info(f"No Corrections introduced in Version {version} were applied.")
+                else:
+                    st.info(f"All Corrections introduced in Version {version} were applied.")
+
+            df = filter_by_version(df, version)
+
+            if df.empty:
+                empty_message(applied, version)
+            else:
+                figure = generate_graph(field_correction_counts(df), version, applied)
+                st.plotly_chart(figure)
+
+                display_corrections_df(df)
+
+
+        st.header("Manual Corrections")
+
+        applied_corrections = data['pluto_corrections_applied']
+        not_applied_corrections = data['pluto_corrections_not_applied']
+
+        if applied_corrections is None or not_applied_corrections is None:
+            st.info("There are no available corrections reports for this branch.")
+            return
```

Comment:
This block doesn't seem necessary to me. Doesn't it make more sense to get the associated table from `get_data` inside the `create_corrections_section` function so we only have to pass one variable? 
Also this approach would lead not visualizing corrections applied if corrections not applied was none and vice versa. 
Also I think we want to check the `.empty` method of the dataframes  instead of whether they are `None`. This check is good, it should just happen in the `create_corrections_section` call

Reply:
This is testing something slightly different than the checks that exist in the create_corrections_section for whether there are no corrections for the version - it's for if the branch itself has no files for the reports in Digital Ocean. I think it might be unnecessary since it's mostly testing for a condition most likely to occur to a developer, but there's no real time at which this 
```
Also this approach would lead not visualizing corrections applied if corrections not applied was none and vice versa.
```
would be relevant, since the files are most likely both there or both not there. 

 I could alternatively catch the no files thing and return an empty dataframe instead of None (see https://github.com/NYCPlanning/data-engineering-qaqc/pull/70/files#diff-57a9a9900497ce261b5bb70460786b26bbc676bca5270cf37efd9da08368de96R108) and let the checks in create_corrections_section handle the nothing to display issue, but that will tell the end-user less about what the actual problem is

Reply:
Ok great! That makes sense. Although I think giving a dictionary a non-existent key would prompt an error instead of returning `None`. The dictionary's `.get` method would return `None` if no such key exists.

I still like passing `create_corrections_section` one variable for whether it's corrections applied or not applied, and setting everything based on that one variable. I know this is quite lightweight but it would make the code more readable